ansible/roles/install_poetry/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+- name: Check cloned git
+  become: true
+  stat:
+    path: /home/{{ user_name }}/git
+  register: repo_result
+- name: Clone git repo
+  become: true
+  git:
+    repo: "{{ git_clone_url }}"
+    dest: /home/{{ user_name }}/git
+    version: "{{ git_branch }}"
+    force: yes
+- name: Poetry project install
```

Comment:
Have an error here in systemd logs
```
Jun 29 10:39:33 tms bash[25768]: Using python3.11 (3.11.4)
Jun 29 10:39:33 tms bash[25768]: Command not found: flask
```
service fails to start

Reply:
Seems like poetry install does not run

Reply:
maybe it was missed for some reason, let's try now